vhdbuilder/packer/components.json

```diff
@@ -306,7 +292,7 @@
         "v1.8.0.2"
       ],
       "multiArchVersions": [
-        "v1.10.0"
```

Comment:
"v1.10.0" is not in use anymore?

Reply:
azuredisk-csi:v1.9.0 is the minimum version with 'multi-arch'. I will use 1.9.0 instead of 1.10.0.  Current RP doesn't use 1.10.0 at all.